src/components/common/playlist/Song.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import {
+	Dimensions,
+	Image,
+	PixelRatio,
+	Pressable,
+	StyleSheet,
+	Text,
+	View
+} from "react-native";
+import PropTypes from "prop-types";
+import { COLOR_PALETTE } from "../../../utils/constants";
+import { handleSelectSong } from "../../../utils/MusicPlayerUtil";
+
+const windowWidth = Dimensions.get("window").width;
+
+const Song = ({ song, index, playlist }) => {
```

Comment:
Let's re-use the song object we already have under `components/song` also, let's move `common/playlist/` archive to the component level, so we have `component/playlist/` hope that makes sense.